src/spectrecoff/Prompt.fs

```diff
@@ -40,7 +46,10 @@ let chooseFrom choices question =
     prompt (Prompts.selectionPrompt question choices)
 
 let chooseMultipleFrom choices question = 
-    prompt (Prompts.multiSelectionPrompt question choices)
+    prompt (Prompts.multiSelectionPrompt question choices defaultMultiSelectionOptions)
+
+let chooseMultipleFromWith (options: MultiSelectionPromptOptions) choices question = 
```

Comment:
This type should be inferable, right?

Reply:
Yeah, the explicit types were a leftover from the brainstorm phase - good catch!